include/glow/ExecutionEngine/ExecutionEngine.h

```diff
@@ -83,6 +83,8 @@ class ExecutionEngine final {
 
   /// Runs a single execution of the function.
   void run();
+  /// Context aware single execution of the function
+  void runctx(Context &ctx);
```

Comment:
why not call this simply `run`?

Reply:
no good reason.. and it won't be hard to fix let me correct that.